tools/pd-ctl/pdctl/command/unsafe_command.go

```diff
@@ -84,25 +75,20 @@ func removeFailedStoresCommandFunc(cmd *cobra.Command, args []string) {
 	postInput := map[string]interface{}{
 		"stores": stores,
 	}
-	postJSON(cmd, prefix, postInput)
-}
-
-func removeFailedStoresShowCommandFunc(cmd *cobra.Command, args []string) {
-	var resp string
-	var err error
-	prefix := fmt.Sprintf("%s/remove-failed-stores/show", unsafePrefix)
-	resp, err = doRequest(cmd, prefix, http.MethodGet, http.Header{})
+	timeout, err := cmd.Flags().GetFloat64("timeout")
 	if err != nil {
-		cmd.Println(err)
+		cmd.Usage()
```

Comment:
Should we print err?

Reply:
yes, changed

Reply:
line 62 also need to be changed

Reply:
I think line 70 should be changed, but not line 62